modules/edu.gemini.seqexec.engine/src/main/scala/edu/gemini/seqexec/engine/model.scala

```diff
@@ -187,7 +201,8 @@ object Queue {
           mseq match {
             // No more Steps in current Sequence, remove Sequence.
             // TODO: listTailPLens?
-            case None => ((exe, seq0.id, stepid).left.left, Queue(q.sequences.tailOption.getOrElse(List())))
+            case None => ((exe, seq0.id, stepid).left.left,
+                          Queue(q.sequences.tailOption.getOrElse(List())))
```

Comment:
Good to know. I think I have more instances of `List()` spread over the code, so I'll change them all.